<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigSoft</class>
 <widget class="QDialog" name="DialogConfigSoft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panneau de configuration</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/img/img/close.png</normaloff>:/img/img/close.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="validate">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Appliquer</string>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/img/img/validate.png</normaloff>:/img/img/validate.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>CNC</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Répertoire LinuxCNC :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Exécutable LinuxCNC :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="linuxcncBin"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="linuxcncDirectory"/>
     </item>
     <item row="0" column="2">
      <widget class="FolderButton" name="folderCNC">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="FolderButton" name="folderExeCNC">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Générale</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>276</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Executable PolyPlexer :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Périphérique Virtuel : </string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="virtualDevice"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="baudrate"/>
     </item>
     <item row="0" column="2">
      <widget class="FolderButton" name="folderHome">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Répertoire périphériques :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="homeDirectory"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="serialDeviceCombo"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Repertoire Home :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Nom périphérique série :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="FolderButton" name="folderExePolyPlexer">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="FolderButton" name="folderVirtDevice">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="serialPortPath"/>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="activateByPass">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Baudrate : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="FolderButton" name="folderDevices">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="polyplexerBin"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Désactiver les contrôles :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Désactiver les contrôles :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="disablePolyplexer">
       <property name="text">
        <string> (Pour les experts)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder sous...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAsDefault">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder défaut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFactoryValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger Valeurs usines</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Imprimante</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="printersoftBin"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Executable :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Répertoire de travail :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="printerworkingdir"/>
     </item>
     <item row="0" column="2">
      <widget class="FolderButton" name="folderExePrinter">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="FolderButton" name="folderPrinterWorkin">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FolderButton</class>
   <extends>QPushButton</extends>
   <header>FolderButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
